com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/PythonTensorAnalysisEngine.java

```diff
@@ -150,7 +150,11 @@ private static Set<PointsToSetVariable> getDataflowSources(Graph<PointsToSetVari
                   || Objects.equal(tensorFlowAPI, "convert_to_tensor")
                   || Objects.equal(tensorFlowAPI, "range")) sources.add(src);
               // Double-level APIs.
-              else if (Objects.equal(tensorFlowAPI, "uniform")) {
+              else if (Objects.equal(tensorFlowAPI, "uniform")
```

Comment:
I feel like this code needs to be in a loop with a variable index or the `level`. There looks like there's too much repetition. But, I wouldn't do this straight-away; it sounds like like we have a bug elsewhere that is affecting this.